config/cp4waiops-custom/values.x-small.yaml

```diff
@@ -267,7 +267,7 @@ res:
     elasticsearch:
       limits:
         cpu: 500m
-        memory: 1536Mi
+        memory: 1535Mi
```

Comment:
Why it's changed from 1536 to 1535? It looks 1536 is a more reasonable value as it aligns with the binary paddings, i.e.: 1536 = 1024 + 512.

Reply:
It is tiny trick with magic number to flag some special and make verification much easier. 
And in this case, 1535 is less then 1536 and will not result extra memory paging waste.